README.md

```diff
@@ -85,7 +85,7 @@ generic with minimal need for project-specific behavior.
 
 ## 2.1 Development Prerequisites
 
-- Python 3.6 (3.7 does not work) with `pip`.
+- Python 3.8 (3.7 does not work) with `pip`.
```

Comment:
```suggestion
- Python 3.8 (no other Python 3.x is supported) with `pip`.
```

Reply:
Going instead with "(no other version is supported)" since this excludes even non 3.x versions.

Reply:
Please use my version.